eng/build.ps1

```diff
@@ -162,9 +259,18 @@ try {
     . $configureToolsetScript
   }
 
+  # IBC merge is only invoked in official build, but we want to enable running IBCMerge locally as well.
```

Comment:
. . . but only when run as CI? That's how I interpret the condition below.

Reply:
I think I fixed it.